src/apps/content-editor/src/app/views/ItemEdit/ItemEdit.js

```diff
@@ -202,6 +205,13 @@ export default function ItemEdit() {
         );
         return;
       }
+
+      // fetch nav only when metaLinkText is updated
+      if (item.web.metaLinkText !== initialNavLink) {
```

Comment:
If that's the case, should I put an extra parameter for the dispatch of saveItem? like `dispatch(saveItem(itemZuid, initialNavLink))` just to fetch the initialNavLink from the component.

Since I think that's the only way that I could get the initial nav link without breaking the pattern of the store.

Reply:
@agalin920 Noted on that